daemon/memcached.c

```diff
@@ -6619,6 +6632,11 @@ static void unregister_extension(extension_type_t type, void *extension)
             EXTENSION_DAEMON_DESCRIPTOR *prev = NULL;
             EXTENSION_DAEMON_DESCRIPTOR *ptr = settings.extensions.daemons;
 
+            if (ptr == NULL) {
+                // No daemon extensions are registered.
```

Comment:
You may want to mention "checking to avoid static analysis issues", as the code itself didn't need this check.

Reply:
@Ying-metaswitch Are you sure? As far as I can tell, there's no reason this couldn't be null, but if you know better I'll change it.

Reply:
Nah I meant the loop below is `while (ptr != NULL)` anyway....It's fine keeping the comment this way.